Cypress/cypress/pages/workflow.js

```diff
@@ -364,4 +360,46 @@ Cypress.Commands.add("validateRecurringStats", () => {
         .eq(2)
         .click();
     });
+});
+
+/// ************************** Validate workflow existence on cluster **********************
+
+Cypress.Commands.add("validateWorkflowExistence", (workflowName, namespace) => {
+  let workflowFound = false;
+  cy.request({
+    url: apis.getWorkflows(namespace),
+    method: "GET",
+    headers: {
+      Authorization: `Bearer ${KUBE_API_TOKEN}`,
+      'Content-Type': 'application/json'
+    },
+  }).should((response) => {
+    response.body.items.some((item) => {
+      if (item.metadata.name === workflowName){
+        workflowFound = true;
+        return true;
+      }
+    });
+    if (workflowFound === false){
+      throw new Error("Workflow Not Found in cluster");
+    }
+  });
+});
+
+/// ************************** Validate workflow existence on cluster **********************
+
+Cypress.Commands.add("validateWorkflowStatus", (workflowName, namespace) => {
```

Comment:
Can we take expected status also as argument & return true or throw error based on equality condition.